### DISCUSSION TITLE
how can i see or get  the current debugged  file path?

### DISCUSSION THREAD

**Discussion by robator0127** [inducer/pudb#641] [Q&A]
when i step into( or out) a function or a class ,the new source code may be in another file ,how can i see or get the whole file path of current debugged file ,as debug in the pdb ?thanks

**Comment by Kevin-Prichard** [comment]
I don't see a way for that to happen currently.  Might make for a nice toggle-able option in the Stack list, similar to how wrap works in the Variables list.

**Comment by antonskal** [comment]
I was trying to see the path's of my modules and for that

print(modulename.__file__)
